tests/release/e2e-test-releaseplan-owner-ref.go

```diff
@@ -42,8 +42,16 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-2469]test-releaseplan-owner-ref-a
 		It("verifies that the ReleasePlan has an owner reference for the application", func() {
 			releasePlan, err = fw.AsKubeAdmin.ReleaseController.GetReleasePlan(sourceReleasePlanName, devNamespace)
 			Expect(err).NotTo(HaveOccurred())
-			ownerRef := releasePlan.OwnerReferences[0]
-			Expect(ownerRef.Name).To(Equal(applicationNameDefault))
+			Eventually(func() error {
+				if len(releasePlan.OwnerReferences) < 1 {
```

Comment:
It should have a single owner reference.
```suggestion
				if len(releasePlan.OwnerReferences) != 1 {
```